apps/core/management/commands/create_test_feature_switches.py

```diff
@@ -14,6 +14,7 @@
     ("show_testclient_link", True),
     ("interim-prod-access", True),
     ("enable_swaggerui", True),
+    ("enable_internal_application_labels", True),
```

Comment:
Since this is an internal only feature with no immediate impact externally, I think I'd prefer not using a switch for this. I think we would just do a migration prior to the code being deployed, and then have it enabled once the code is deployed. I think it would make a lot of the other code in this PR simpler too.